src/components/product/ProductCard.tsx

```diff
@@ -0,0 +1,79 @@
+import styled from "styled-components";
+import { Product } from "../../types/Product";
+import { CardButton } from "./CardButton";
+import { Suspense } from "react";
```

Comment:
`eslint`를 이용해서 `import` 정렬을 해보아도 좋을 것 같습니다!

```suggestion
import styled from "styled-components";
import type { Product } from "../../types/Product";
import { CardButton } from "./CardButton";
import { Suspense } from "react";
```

그리고 타입에는 `type`을 명시하면 좋을 것 같습니다.